src/constants.ts

```diff
@@ -17,13 +20,65 @@ export const REPORT_ERROR_FORM_ENTRIES = {
 export const REPORT_ERROR_FORM_URL =
   "https://docs.google.com/forms/d/1YOvhK2RfTQmYO8DGMRqN7FYxRhBZd9jB6PZ7InJirTk/viewform";
 
-export const USDT_DECIMALS = 6;
-
 export const UNISWAP_V3_POOL_FEE = 3000;
 
 export const FIAT_EXCHANGE_RATES_API_URL = "https://api.exchangeratesapi.io/v1/latest";
 
-export const chains: Chain[] = [
-  { name: "Ethereum chain", icon: EthChainIcon, chainId: 1 },
-  { name: "Polygon Hermez chain", icon: PolygonHermezChainIcon, chainId: 10 },
-];
+export const CHAINS: Record<SupportedNetwork, Chain[]> = {
+  [SupportedNetwork.LOCAL]: [
+    {
+      name: "Ethereum chain",
+      Icon: EthChainIcon,
+      chainId: 1337,
+    },
+    {
+      name: "Polygon Hermez chain",
+      Icon: PolygonHermezChainIcon,
+      chainId: 1001,
+    },
+  ],
+};
+
+export const BRIDGE_CONFIGS: Record<SupportedNetwork, BridgeConfig> = {
+  [SupportedNetwork.LOCAL]: {
+    l1ContractAddress: "0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9",
+    l2ContractAddress: "0x9d98deabc42dd696deb9e40b4f1cab7ddbf55988",
+    apiUrl: "http://localhost:8080",
+  },
+};
+
+export const UNISWAP_QUOTER_ADDRESSES: Record<SupportedNetwork, string> = {
+  [SupportedNetwork.LOCAL]: "0xb27308f9F90D607463bb33eA1BeBb41C27CE5AB6",
+};
+
+export const L1_PROVIDERS: Record<SupportedNetwork, string> = {
+  [SupportedNetwork.LOCAL]: "http://localhost:8123",
+};
+
+export const L2_PROVIDERS: Record<SupportedNetwork, string> = {
+  [SupportedNetwork.LOCAL]: "http://localhost:8545",
+};
+
+export const ETH_TOKENS: Record<SupportedNetwork, Token> = {
+  [SupportedNetwork.LOCAL]: {
+    name: "Ether",
+    address: ethers.constants.AddressZero,
+    symbol: "ETH",
+    decimals: 18,
+    chainId: 1337,
+    logoURI:
+      "https://raw.githubusercontent.com/Uniswap/interface/main/src/assets/images/ethereum-logo.png",
+  },
+};
+
+export const USDT_TOKENS: Record<SupportedNetwork, Token> = {
+  [SupportedNetwork.LOCAL]: {
+    name: "Tether USD",
+    address: "0xdAC17F958D2ee523a2206206994597C13D831ec7",
+    symbol: "USDT",
+    decimals: 6,
+    chainId: 1,
+    logoURI:
+      "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0xdAC17F958D2ee523a2206206994597C13D831ec7/logo.png",
+  },
+};
```

Comment:
Why did you decide to split each config and then group it in a record by the network? I see low cohesion in the current implementation.

All this feels better organized by defining a `Network` interface, so when you have a `Network` you get all its data at once:

```typescript
type Network = {
  type: SupportedNetwork;
  l1Chain: Chain;
  l2Chain: Chain;
  bridge: BridgeConfig,
  uniswapQuoterAddress: string;
  ethToken: Token;
  usdToken: Token;
};
```